include/net/ethernet.h

```diff
@@ -81,6 +81,9 @@ enum ethernet_hw_caps {
 
 	/** Promiscuous mode supported */
 	ETHERNET_PROMISC_MODE		= BIT(10),
+
+	/** Priority queues available */
+	ETHERNET_PRIORITY_QUEUES	= BIT(11),
```

Comment:
Just a note that this will conflict with #9153 PR as it is adding a bit here too.